hadoop-hdds/rocks-native/pom.xml

```diff
@@ -235,6 +235,7 @@
                       <arg line="${project.build.directory}/rocksdb/rocksdb-${rocksdb.version}"/>
                     </exec>
                     <exec executable="make" dir="${project.build.directory}/rocksdb/rocksdb-${rocksdb.version}" failonerror="true">
+                      <arg line="PORTABLE=1"/>
```

Comment:
RocksDB does not use PORTABLE=0 so there is still some information missing as to what is the build setup.
For now pushing this in makes sense as we are seeing crashes for missing AVX4 but we still need to continue looking into what is the lowest common set of instruction set the build should support or at least quantify the performance impact. @swamirishi can you open a follow up Jira? Also, how was this change tested?

Reply:
The `.so` from the publicly distributed `rocksdbjni-7.7.3.jar` doesn't seem to be AVX or AVX2 optimized:

```bash
$ elfx86exts librocksdbjni-linux64.so 
File format and CPU architecture: Elf, X86_64
MODE64 (call)
CMOV (cmovbe)
SSE1 (movups)
SSE2 (movdqa)
BMI (tzcnt)
BMI2 (shlx)
Instruction set extensions used: BMI, BMI2, CMOV, MODE64, SSE1, SSE2
CPU Generation: Haswell
```